packages/proto-signing/src/signing.ts

```diff
@@ -1,8 +1,24 @@
 /* eslint-disable @typescript-eslint/naming-convention */
+import { omitDefaults } from "./adr27";
 import { cosmos } from "./generated/codecimpl";
 
 const { SignDoc } = cosmos.tx;
 
-export function makeSignBytes(signDoc: cosmos.tx.ISignDoc): Uint8Array {
+export function makeSignBytes(
```

Comment:
This is super helpful. Then we just need helpers to build `txBody` and `authInfo`